input/otlp/logs.go

```diff
@@ -48,6 +49,11 @@ import (
 	"github.com/elastic/apm-data/model/modelpb"
 )
 
+const (
+	MaxDataStreamRunes        = 100
```

Comment:
nit: do we want these internal naming check constants be exposed from this package?